stdlib/dynarray.mli

```diff
@@ -308,6 +308,46 @@ val filter_map : ('a -> 'b option) -> 'a t -> 'b t
     ignoring strings that cannot be converted to integers.
 *)
 
+val mem : 'a -> 'a t -> bool
```

Comment:
For the documentation part, I would propose to mirror the `Array` documentation by moving `forall` and `exists`  just before `mem` and adding a
```
{1:dynarray_scanning Dynarray scanning }
```
header.